metis-dereference/metis-dereference-service/src/test/java/eu/europeana/metis/dereference/service/MongoDereferencingManagementServiceTest.java

```diff
@@ -1,4 +1,4 @@
-package eu.europeana.metis.derefrence.test;
```

Comment:
Not related to the pr but on line 149
        Assert.assertEquals(entity1.getXml(),"testXml2");
Switch the parameters, first expected then actual.

Reply:
Fixed.